src/app/wizard/step/provider-settings/provider/extended/gcp/component.ts

```diff
@@ -151,13 +163,18 @@ export class GCPProviderExtendedComponent extends BaseFormValidator implements O
       case Controls.SubNetwork:
         return this._hasRequiredCredentials() ? '' : 'Please enter your credentials first.';
     }
+    return null;
```

Comment:
Move it to default inside switch and look on my comment above regarding typings.